src/interfaces.tsx

```diff
@@ -0,0 +1,15 @@
+export interface TeamData {
```

Comment:
I think we could set this up as a type

`export type TeamData = {
  SeasonType: number;
  Season: number;
  Conference: string;
  Division: string;
  Team: string;
  Name: string;
  Wins: number;
  Losses: number;
  Ties: number;
  NetPoints: number;
  Touchdowns: number;
  DivisionWins: number;
  TeamID: number;
}`

Reply:
I have changed it to a type. How do you decide to use a type or interface?
Is it strictly based on inheritance??